packages/mux-player/src/template.ts

```diff
@@ -105,21 +141,24 @@ export const content = (props: MuxTemplateProps) => html`
           : html``
       }
     </mux-video>
-    <button
-      slot="seek-to-live-button"
-      part="seek-live button"
-      class="mxp-seek-to-live-button"
-      aria-disabled="${props.inLiveWindow}"
-      onclick="${function (this: HTMLButtonElement, evt: Event) {
-        props.onSeekToLive?.(evt);
-        if (props.paused) {
-          const playRequestEvt = new CustomEvent('mediaplayrequest', { composed: true, bubbles: true });
-          (this as HTMLButtonElement).dispatchEvent(playRequestEvt);
-        }
-      }}"
-    >
-      Live
-    </button>
+    ${
```

Comment:
question(non-blocker): Why was this refactored? Was slotting this causing problems (It shouldn't have)? If not, I'd lean towards always slotting, since it's less presumptuous.

Reply:
`part="${isLive(props) ? 'top' : 'bottom'} seek-live button"` this is the needed change and I noticed this is only required for live like streams so it made sense to me to not render this for VOD. no big changes in here though, just added 2 conditionals

Reply:
hrm. Did you not change the rendering logic to conditionally render based on live vs. not? Currently, that shouldn't have any impact either way, but it doesn't look like it was necessary.

Reply:
aka
* before: always rendered the seek live `<button>` and let the theme figure out if it "cared" about the slotted element "under the hood"
* after: conditionally render the seek live `<button>` "from the outside"

Reply:
yes, in the case of VOD we won't ever need a seek to live button so it shouldn't be rendered at all. 
I should've done this from the start but I didn't realize it at the time.

Reply:
not going to push back here as a blocker to this PR, but there are some benefits to always passing in slotted elements as we start having robust support of themes, since that lets theme implementors decide whether or not they want to use the slotted element and under what conditions.